drivers/ipm/ipm_imx.c

```diff
@@ -15,6 +15,9 @@
 #if defined(CONFIG_IPM_IMX_REV2)
 #define DT_DRV_COMPAT nxp_imx_mu_rev2
 #include "fsl_mu.h"
+#undef DT_DRV_COMPAT
+#define DT_DRV_COMPAT nxp_imx_mu_dsp
```

Comment:
Why use a different compatible here? It seems the same header is being included

Reply:
As mentioned in the commit message, I added a new compatible in order to skip the RDC dependency from mailbox node. 
Now, the mailbox node, with `nxp_imx_mu_rev2` compatible has the RDC property mandatory, and for DSP we don't use/we don't need the RDC.

Reply:
Sorry, I should have caught this in the commit message. Rather than introduce a new compatible, why not either:
- make the `rdc` field optional for this compatible?
- include the `rdc` property, and set it to a dummy value (this is what we do on the RT1170 series)

Reply:
> Sorry, I should have caught this in the commit message. Rather than introduce a new compatible, why not either:
> 
> * make the `rdc` field optional for this compatible?
> * include the `rdc` property, and set it to a dummy value (this is what we do on the RT1170 series)

Thank you @danieldegrasse!
I've looked on RT1170 ([here](https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/arm/nxp/nxp_rt11xx_cm7.dtsi#L83)) and I can use `rdc = <0>` for my case too.

Reply:
Fixed: removed the new compatible and added a dummy value for RDC.